### PR TITLE
[WIP] Onnx export API optional args and additional operator support, Fixes #12682

### LINKED ISSUE #12682: No conversion function registered for op type BlockGrad yet
When I transform my mxnet model to onnx model, I met the following error:
image
Here is my *-symbol.json file:
image

### PR DESCRIPTION
**Pull request by Roshrini**
## Description ##
ONNX export api takes optional arguments out_label and out_shapes
Adding operators - LogisticRegressionOutput, MakeLoss, BlockGrad, _rminus_scalar, r_div_scalar, identity

https://github.com/apache/incubator-mxnet/issues/12925

WIP - Need to add:
- Test Cases for additional operators
- Test case for multiple output model
- Add comments

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by Roshrini** [comment]
@mxnet-label-bot [ONNX, pr-work-in-progress]

**Comment by vandanavk** [comment]
@Roshrini please add 

> Fixes https://github.com/apache/incubator-mxnet/issues/12682

 in the PR description to automatically close the issue when PR is merged.

**Comment by kalyc** [comment]
@Roshrini thanks for you contribution. There appears to be build failure could you take a look?

**Comment by stu1130** [comment]
@Roshrini ping again to address CI and description

File: python/mxnet/contrib/onnx/mx2onnx/_op_translations.py [hunk 1]
```diff
@@ -926,8 +966,14 @@ def convert_minus_scalar(node, **kwargs):
     """
     return scalar_op_helper(node, 'Sub', **kwargs)
 
+@mx_op.register("_rminus_scalar")
+def convert_rminus_scalar(node, **kwargs):
+    """Map MXNet's _rminus_scalar operator attributes to onnx's Minus operator.
```
Comment (vandanavk) [inline python/mxnet/contrib/onnx/mx2onnx/_op_translations.py]: nit: onnx's Sub operator

**Comment by Roshrini** [comment]
Will update and create new PR

**Pull request closed without merging**